engine/src/main/java/org/terasology/engine/core/module/ModuleManager.java

```diff
@@ -110,27 +125,6 @@ private void loadModulesFromApplicationPath(PathManager pathManager) {
         scanner.scan(registry, paths);
     }
 
-    private Module loadEngineModule(List<Class<?>> classesOnClasspathsToAddToEngine) {
-        // FIXME: is `classes…toAddToEngine` gone? Did we ever use it in the first place?
```

Comment:
Did you check for `classesOnClasspathsToAddToEngine` usage? I know it's not usually used, so maybe this is a non-issue, but I had a vague memory about it being used somewhere...